### PR TITLE
Reservations

### PR DESCRIPTION
**Pull request by kybak**

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -29,8 +28,10 @@
     "styled-components": "^4.1.3"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build-semantic": "cd src/semantic && gulp build",
+    "watch-semantic": "cd src/semantic && npm run build-semantic && gulp watch",
+    "start": "npm run build-semantic && react-scripts start",
```
Comment (LBWright) [inline client/package.json:33]: cool

File: client/src/Components/Dashboard/container.js [hunk 2]
```diff
@@ -6,10 +6,6 @@ const mapStateToProps = ({ registration }) => ({
   registration
 });
 
-const mapDispatchToProps = () => ({
```
Comment (LBWright) [inline client/src/Components/Dashboard/container.js:9]: With mapDispatchToProps, you can either do it as an object, or as a function like this (but you have to import bindActionCreators from react-redux). But what you have is obviously acceptable. For more actionCreators though, it's worth considering.
```
const mapDispatchToProps = (dispatch) => bindActionCreators({ registerUser: registerUser }, dispatch)
```

File: client/src/Components/Reservations/actionCreator.js [hunk 3]
```diff
@@ -38,3 +38,44 @@ export const searchReservations = (filterSort = {}) => dispatch => {
...
+    });
+};
+
+export const fetchEmployees = () => dispatch => {
+  dispatch({ type: actions.RESERVATION_STARTED });
+
+  return axios
+    .get(`${config.apiUrl}/api/employees`)
+    .then(({ data }) => {
+      dispatch({ type: actions.EMPLOYEES_SUCCESS, employees: data.data });
```
Comment (LBWright) [inline client/src/Components/Reservations/actionCreator.js]: Usually, you want the 2nd property in your action to be payload. Each action should have a type and a payload.

File: client/src/Components/Reservations/reducer.js [hunk 4]
```diff
@@ -19,6 +21,18 @@ const reservationsReducer = (state = initialState, action) => {
         loading: false,
         reservations: action.reservations
       };
+    case actions.PROPERTIES_SUCCESS:
+      return {
+        ...state,
+        loading: false,
+        properties: action.properties
```
Comment (LBWright) [inline client/src/Components/Reservations/reducer.js]: You use the payload variable name to keep consistency here. You don't have to remember what you named the action's payload.
You can also destructure the action parameter if you do that.
```
const reducer = (state = initialState, {type, payload}) => {...}
```

**Review by jessharrison83 — APPROVED** [review]

**Pull request merged**